client/src/app/shared/components/icon-container/icon-container.component.scss

```diff
@@ -0,0 +1,39 @@
+@import '~@angular/material/theming';
```

Comment:
Why do you use import of theming in a scss file which does not use theme colors?